descqa/shear_test.py

```diff
@@ -206,26 +211,26 @@ def get_catalog_data(gc, quantities, filters=None):
 
     # define theory from within this class
 
-    def post_process_plot(self, ax):
+    def post_process_plot(self, ax, fig):
```

Comment:
To maintain compatibility with other calls of this function..
```suggestion
    def post_process_plot(self, ax, fig=None):
```

Reply:
I already caught this one myself and fixed it.